version/constants.go

```diff
@@ -0,0 +1,6 @@
+package version
+
+var (
+	RosettaVersion = "v0.1.0"
```

Comment:
a lot of exported items in the branch that don't have a comment

Reply:
Added extra few (not all).